### ISSUE TITLE
Please add audio recording function and fix the flash while recording, the autoflash or manual flash for video recording doesn't work

### ISSUE THREAD

**Issue by User1l0** [hardcore-sushi/DroidFS#60]

**Comment by hardcore-sushi** [comment]
Yes, I could add flash control feature.
By audio recording you mean audio only recording?

**Comment by User1l0** [comment]
> Yes, I could add flash control feature. By audio recording you mean audio only recording?

Yes, that you can record audios, bet i lt will be nice, if you add quality options to, to select the kbs and audio format like wav mp4 m4a and more

**Comment by hardcore-sushi** [comment]
OK but it would require to enable more features in FFmpeg so it will increase the final APK size. Moreover, that's not the main purpose of DroidFS. If I add a audio-only recording feature, I think it will be very simple, and will probably uses the same codec as for video recording.
That's not the priority, but pull requests are welcome!

**Comment by User1l0** [comment]
> OK but it would require to enable more features in FFmpeg so it will increase the final APK size. Moreover, that's not the main purpose of DroidFS. If I add a audio-only recording feature, I think it will be very simple, and will probably uses the same codec as for video recording. That's not the priority, but pull requests are welcome!

Audio recording function would be very nice, try to make it use mp4 codec to make m4a audio files, it is also a great audio format

**Comment by hardcore-sushi** [comment]
Timer and flash have been fixed for video recording: b65ee230beb47d986259f54a5b4f6941ea6765b5. The fixes will be included in the next DroidFS release.

**Comment by OmlineEditor** [comment]
now there is a video recording with sound. just record audio with the same codec without video
